Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang:1.16 AS mod_cacher
+
+WORKDIR /app
+
+COPY go.mod go.sum /app/
+RUN go mod download
+
+FROM golang:1.16 AS builder
+
+COPY --from=mod_cacher $GOPATH/pkg/mod $GOPATH/pkg/mod
+COPY . "${GOPATH}/src/github.com/crypto-com/chain-indexing"
+RUN cd "${GOPATH}/src/github.com/crypto-com/chain-indexing" \
+    && make chain-indexing \
+    && go get -tags 'postgres' github.com/golang-migrate/migrate/v4/cmd/migrate
+
+FROM golang:1.16
+
+WORKDIR /app
+
+COPY --from=builder "${GOPATH}/bin/migrate" "${GOPATH}/bin/"
+COPY --from=builder "${GOPATH}/src/github.com/crypto-com/chain-indexing/chain-indexing" .
+COPY ./migrations ./migrations
+COPY ./pgmigrate.sh ./migrate
+
+EXPOSE 28857
```

Comment:
I am not sure about the common practice, but should we make this port exposure to be an runtime ENV? @davcrypto  maybe can comment.

At the bottom line, since our configuration are using `8080` as the default, it will be desire-able to use it here.

Reply:
Or we can do `-p` to specify the port(s) to be exposed in `docker run` command @calvinaco

Reply:
rather in here